mautic/mautic
# M3: Refactor Deprecations - MauticCitrixBundle

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | https://github.com/mautic/mautic/issues/8015
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR refactors Mautic Citrix bundle. It should be compatible with Symfony 3.

#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. You can create a trial gotowebinar account at their website. It's good for 7 days.
3. Setup and authenticate with Gotowebinar.
4. Setup a campaign and add the "Gotowebinar action"
5. You should see Sample Webinar listed.
